lib/wallet/wallet.js

```diff
@@ -1844,6 +1848,10 @@ class Wallet extends EventEmitter {
   async _sendOpen(name, force, options) {
     const passphrase = options ? options.passphrase : null;
     const mtx = await this._createOpen(name, force, options);
+
+    if (options && options.signal && options.signal.aborted)
+      throw new common.AbortError('operation was aborted.');
```

Comment:
```suggestion
    if (options?.signal?.aborted)
      throw new common.AbortError();
```

Same for all other ocurrency.